source/developer-guides/zephyr-with-sbl.rst

```diff
@@ -0,0 +1,115 @@
+.. _zephyr-with-sbl:
+
+Zephyr with |SPN| 
+-----------------
+
+We may follow below steps to setup Zephyr with |SPN| on |UP2| maker board and QEMU.
+
+|UP2| maker board
+^^^^^^^^^^^^^^^^^
+
+1. Follow the '|Getting Started Guide|' to setup **Zephyr** build/dev environment
+
+.. |Getting Started Guide| raw:: html
+
+   <a href="https://docs.zephyrproject.org/latest/getting_started/getting_started.html" target="_blank">Getting Started Guide</a>
+
+2. Apply a patch for testing |SPN| with |UP2| maker board:
+
+  1.) Apply a simple patch to fix-up peripheral mmio with |PN|::
+  
+        git apply xxxx.patch
+        
+        or git am xxxx.patch
+        
+        or patch -p1 < xxxx.patch
+
+  2.) There are known warnings coming from the simple patch::
+  
+        simple-bus unit address format error
+
+3. Build an application uses synchronization sample application:
+
+  1.) setup zephyr environment::
+  
+        source zephyr-env.sh
+        or source zephyr-env.cmd
+
+  2.) Build synchronization for |UP2| board::
+  
+        cd samples/synchronization
+        mkdir build_up2 && cd build_up2
+        cmake -GNinja -DBOARD=up_squared ..
```

Comment:
Use up_squared_sbl instead of up_squared
cmake -GNinja -DBOARD=up_squared_sbl ..